MEAnalysis/crab_vhbb/heppy_crab_script.py

```diff
@@ -44,8 +44,11 @@ def getLumisProcessed(vlumiblock):
     if arg.startswith("ME_CONF="):
         me_conf_name = arg.split("=")[1]
 
-JobNumber=sys.argv[1]
-print "JobNumber", JobNumber
+try:
```

Comment:
It's this:
https://github.com/jpata/tthbb13/blob/meanalysis-80x/MEAnalysis/crab_vhbb/heppy_crab_script.py#L9
so in this case, we configure everything local.
In any case, we're not using `JobNumber`, so I would just remove all lines related to it and make the code simpler.